[DOCS-6574] New Resource Catalog page (#20423)
DataDog/documentation: content/en/infrastructure/resource_catalog/_index.md

---
title: Datadog Resource Catalog
kind: documentation
is_beta: true
aliases:
  - /security_platform/cspm/resource_catalog
  - /security/cspm/resource_catalog
  - /security/misconfigurations/resource_catalog
further_reading:

> - link: "security/misconfigurations/"

Reviewer: ```suggestion
- link: "/security/misconfigurations/"
```

  tag: "Documentation"
  text: "Cloud Security Management Misconfigurations"

> - link: "security/threats/"

Reviewer: ```suggestion
- link: "/security/threats/"
```

  tag: "Documentation"
  text: "Cloud Security Management Threats"
---

{{< site-region region="gov" >}}
<div class="alert alert-warning">
Resource Catalog is not available in this site.
</div>
{{< /site-region >}}

<div class="alert alert-info">Resource Catalog is in beta.</div>

## Overview

Datadog Resource Catalog provides a high-level overview of the hosts and resources in your cloud and hybrid environments. View information such as tags, configuration details, relationships between assets, misconfigurations, and threats. See what team is responsible for each resource, and what security misconfigurations have been reported. Access dashboards and Datadog views that receive and monitor telemetry and security data for each resource.

Resource Catalog leverages Datadog cloud integrations and the Datadog Agent to gather data from cloud resources such as databases, storage services, and hosts.

{{< img src="/infrastructure/resource_catalog/Resource_Catalog.png" alt="The Resource Catalog page showing the Inventory tab, sorting by service" width="100%">}}

## Setup

> By default, you can view all resources monitored by the Datadog Agent and other Cloud Integrations. To view additional cloud resources in the Resource Catalog, please enable resource collection from the [Resource Catalog][5] setup page. To gain insights into your security risks, enable [Cloud Security Management][1] for each cloud account.

Reviewer: ```suggestion
By default, you can view all resources monitored by the Datadog Agent and Cloud Integrations. To view additional cloud resources in the Resource Catalog, enable resource collection from the [Resource Catalog][5] setup page. To gain insights into your security risks, enable [Cloud Security Management][1] for each cloud account.
```

On the [Resource Catalog page][2], explore the cloud resources in your Datadog organization. The catalog detects a resource either because it has an Agent installed on it, or because a cloud integration is configured on it. Information about the resources in your organization is shown in the ownership and security tabs, with two views: List and Map.

**Inventory Tab**:

> The inventory tab allows you to understand the context for a resource such as team ownership and related service. This page allows you to proactively identify lack of ownership information and resolve it prior to an incident occurring. The Resource Catalog presents configuration information customized to the resource type being viewed. It also allows you to search resources by specific configuration attributes such as `instance_type` for a host, or a `version` for a database.

Reviewer: ```suggestion
The Inventory tab shows context for a resource, including team ownership and related services. It helps you proactively identify and provide missing ownership information before you need it in an incident. The Resource Catalog shows configuration information customized for each resource type. You can search resources by specific configuration attributes such as `instance_type` for a host, or a `version` for a database.
```
